gatsby/create-pages/create-redirects.ts

```diff
@@ -156,16 +180,25 @@ export const createRedirects = async ({ actions }: CreatePagesArgs) => {
 
     // the redirect only works for exact matches, that's why we also need to
     // create a redirect without the trailing slash
-    const additionalFromPath = redirect.fromPath.substring(
+    const fromPathWithoutTrailingSlash = redirect.fromPath.substring(
       0,
       redirect.fromPath.length - 1,
     );
 
     createRedirect({
-      fromPath: additionalFromPath,
+      fromPath: fromPathWithoutTrailingSlash,
       toPath: redirect.toPath,
       isPermanent: true,
       redirectInBrowser: true,
     });
   });
+
+  EXTERNAL_REDIRECTS.forEach((redirect) => {
+    createRedirect({
+      fromPath: redirect.fromPath,
+      toPath: redirect.toUrl,
```

Comment:
On my local machine, the redirect was working to my surprise. I always thought that external URLs don't work. Pretty sure it also works on Gatsby Cloud then.